tokio/src/runtime/builder.rs

```diff
@@ -237,9 +237,10 @@ impl Builder {
     /// rt.block_on(async move {});
     /// ```
     ///
-    /// # Panic
+    /// # Panics
     ///
     /// This will panic if `val` is not larger than `0`.
+    #[track_caller]
     pub fn worker_threads(&mut self, val: usize) -> &mut Self {
```

Comment:
Wait, does this function have _two_ sections describing its panics?

Reply:
Funny you should ask... the first Panic section (as far as I can tell) isn't actually true. Calling `worker_threads` for a `current_thread` runtime doesn't panic, at least not immediately and not after scheduling a task on the runtime either.

I was going to submit an issue for this to get some guidance on the preferred behaviour (fix docs or fix code).

Reply:
Created issue for it not panicking in the way described by the first Panic section: #4773